### PR TITLE
AudioPlayer - Enable/Disable all music + sound

### PR DESCRIPTION
**Pull request by NickTUD**
The audioplayer is added so the user can enable / disable all sound with just the press of a button in the app.

As far as I checked, there are no checkstyle warnings what so ever. Also, the cobertura report:
Cobertura report audio

### CODE REVIEW

File: core/src/cg/group4/game_logic/StandUp.java [hunk 1]
```diff
@@ -29,7 +31,12 @@
     /**
      * Singleton of game logic handler.
      */
-    protected static StandUp INSTANCE = new StandUp();
+    protected static StandUp instance = new StandUp();
```
Comment (Surskit) [inline core/src/cg/group4/game_logic/StandUp.java]: Given that it is a singelton, all caps name giving is better.

File: core/src/cg/group4/game_logic/StandUp.java [hunk 2]
```diff
@@ -72,7 +84,7 @@ protected StandUp() {
      * @return INSTANCE
      */
     public static StandUp getInstance() {
-        return INSTANCE;
+        return instance;
```
Comment (Surskit) [inline core/src/cg/group4/game_logic/StandUp.java]: Given that it is a singelton, all caps name giving is better.

**Comment by Surskit** [comment]
Works as expected. +1

**Comment by Surskit** [comment]
WAIT, you are trying to merge into master instead of dev with this pull request.

**Comment by NickTUD** [comment]
Woops

**Pull request closed without merging**